From 0067bf6c5b551a79d6ac3785be756567235347bd Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 10 Oct 2019 11:26:16 +0800
Subject: [PATCH] arm64: dts: rockchip: ramoops use DT reserved-memory bindings

According to commit 529182e204db ("ramoops: use DT reserved-memory bindings")

Change-Id: I73efa992937e6faab5cb2987007d98c8e8294cb0
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 .../arm64/boot/dts/rockchip/px30-android.dtsi | 21 +++++++----------
 arch/arm64/boot/dts/rockchip/px30-robot.dtsi  | 16 +++++--------
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi  | 16 +++++--------
 .../rk3308-voice-module-board-v10.dts         |  9 +++-----
 arch/arm64/boot/dts/rockchip/rk3308.dtsi      | 23 ++++++++-----------
 .../boot/dts/rockchip/rk3328-android.dtsi     | 22 ++++++++----------
 .../boot/dts/rockchip/rk3368-android.dtsi     | 23 ++++++++-----------
 arch/arm64/boot/dts/rockchip/rk3368-xikp.dtsi | 14 -----------
 .../boot/dts/rockchip/rk3399-android.dtsi     | 23 ++++++++-----------
 .../dts/rockchip/rk3399-firefly-android.dts   | 23 ++++++++-----------
 10 files changed, 68 insertions(+), 122 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30-android.dtsi b/arch/arm64/boot/dts/rockchip/px30-android.dtsi
index 7e5ab98850d6..a6aab63213ee 100644
--- a/arch/arm64/boot/dts/rockchip/px30-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30-android.dtsi
@@ -35,20 +35,15 @@
 			compatible = "rockchip,drm-logo";
 			reg = <0x0 0x0 0x0 0x0>;
 		};
-	};
-
-	ramoops_mem: ramoops_mem@110000 {
-		reg = <0x0 0x110000 0x0 0xf0000>;
-		reg-names = "ramoops_mem";
-	};
 
-	ramoops {
-		compatible = "ramoops";
-		record-size = <0x0 0x20000>;
-		console-size = <0x0 0x80000>;
-		ftrace-size = <0x0 0x00000>;
-		pmsg-size = <0x0 0x50000>;
-		memory-region = <&ramoops_mem>;
+		ramoops: ramoops@110000 {
+			compatible = "ramoops";
+			reg = <0x0 0x110000 0x0 0xf0000>;
+			record-size = <0x20000>;
+			console-size = <0x80000>;
+			ftrace-size = <0x00000>;
+			pmsg-size = <0x50000>;
+		};
 	};
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/px30-robot.dtsi b/arch/arm64/boot/dts/rockchip/px30-robot.dtsi
index d3500b92fdf1..abe1b1b73376 100644
--- a/arch/arm64/boot/dts/rockchip/px30-robot.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30-robot.dtsi
@@ -25,22 +25,18 @@
 		status = "okay";
 	};
 
-	ramoops {
-		compatible = "ramoops";
-		record-size = <0x0 0x20000>;
-		console-size = <0x0 0x80000>;
-		ftrace-size = <0x0 0x00000>;
-		pmsg-size = <0x0 0x00000>;
-		memory-region = <&ramoops_mem>;
-	};
-
 	reserved-memory {
 		#address-cells = <2>;
 		#size-cells = <2>;
 		ranges;
 
-		ramoops_mem: ramoops@00000000 {
+		ramoops: ramoops@8000000 {
+			compatible = "ramoops";
 			reg = <0x0 0x8000000 0x0 0xa0000>;
+			record-size = <0x20000>;
+			console-size = <0x80000>;
+			ftrace-size = <0x00000>;
+			pmsg-size = <0x00000>;
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index daf5381bf8d9..819bdc37ed61 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -117,20 +117,16 @@
 			reg = <0x0 0x0 0x0 0x0>;
 		};
 
-		ramoops_mem: region@110000 {
+		ramoops: ramoops@110000 {
+			compatible = "ramoops";
 			reg = <0x0 0x110000 0x0 0xf0000>;
+			record-size = <0x30000>;
+			console-size = <0xc0000>;
+			ftrace-size = <0x00000>;
+			pmsg-size = <0x00000>;
 		};
 	};
 
-	ramoops: ramoops {
-		compatible = "ramoops";
-		record-size = <0x0 0x30000>;
-		console-size = <0x0 0xc0000>;
-		ftrace-size = <0x0 0x00000>;
-		pmsg-size = <0x0 0x00000>;
-		memory-region = <&ramoops_mem>;
-	};
-
 	rk809_sound: rk809-sound {
 		status = "disabled";
 		compatible = "simple-audio-card";
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-voice-module-board-v10.dts b/arch/arm64/boot/dts/rockchip/rk3308-voice-module-board-v10.dts
index 9f94c3579ff7..2586d9905072 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-voice-module-board-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-voice-module-board-v10.dts
@@ -12,11 +12,8 @@
 	compatible = "rockchip,rk3308-voice-module-board-v10", "rockchip,rk3308";
 };
 
-&ramoops_mem {
-	reg = <0x0 0x110000 0x0 0xf0000>;
-};
-
 &ramoops {
-	record-size = <0x0 0x30000>;
-	console-size = <0x0 0xc0000>;
+	reg = <0x0 0x110000 0x0 0xf0000>;
+	record-size = <0x30000>;
+	console-size = <0xc0000>;
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk3308.dtsi b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
index b74116a9596b..afcf01e0da19 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
@@ -278,20 +278,6 @@
 		method = "smc";
 	};
 
-	ramoops_mem: ramoops_mem {
-		reg = <0x0 0x110000 0x0 0xf0000>;
-		reg-names = "ramoops_mem";
-	};
-
-	ramoops: ramoops {
-		compatible = "ramoops";
-		record-size = <0x0 0x30000>;
-		console-size = <0x0 0xc0000>;
-		ftrace-size = <0x0 0x00000>;
-		pmsg-size = <0x0 0x00000>;
-		memory-region = <&ramoops_mem>;
-	};
-
 	rgb: rgb {
 		compatible = "rockchip,rk3308-rgb";
 		status = "disabled";
@@ -326,6 +312,15 @@
 			compatible = "rockchip,drm-logo";
 			reg = <0x0 0x0 0x0 0x0>;
 		};
+
+		ramoops: ramoops@110000 {
+			compatible = "ramoops";
+			reg = <0x0 0x110000 0x0 0xf0000>;
+			record-size = <0x30000>;
+			console-size = <0xc0000>;
+			ftrace-size = <0x00000>;
+			pmsg-size = <0x00000>;
+		};
 	};
 
 	rockchip_suspend: rockchip-suspend {
diff --git a/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi
index 141c254c8788..5bc692b4410d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi
@@ -35,15 +35,20 @@
 			reg = <0x0 0x0 0x0 0x0>;
 		};
 
+		ramoops: ramoops@110000 {
+			compatible = "ramoops";
+			reg = <0x0 0x110000 0x0 0xf0000>;
+			record-size = <0x20000>;
+			console-size = <0x80000>;
+			ftrace-size = <0x00000>;
+			pmsg-size = <0x50000>;
+		};
+
 		secure_memory: secure-memory@20000000 {
 			compatible = "rockchip,secure-memory";
 			reg = <0x0 0x20000000 0x0 0x0>;
 		};
 
-		ramoops_mem: ramoops@68000000 {
-			reg = <0x0 0x110000 0x0 0xf0000>;
-		};
-
 		/* global autoconfigured region for contiguous allocations */
 		linux,cma {
 			compatible = "shared-dma-pool";
@@ -52,15 +57,6 @@
 			linux,cma-default;
 		};
 	};
-
-	ramoops {
-		compatible = "ramoops";
-		record-size = <0x0 0x20000>;
-		console-size = <0x0 0x80000>;
-		ftrace-size = <0x0 0x00000>;
-		pmsg-size = <0x0 0x50000>;
-		memory-region = <&ramoops_mem>;
-	};
 };
 
 &display_subsystem {
diff --git a/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi
index 098b6774d47c..7eac6431d321 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi
@@ -56,20 +56,6 @@
 		interrupts = <GIC_SPI 154 IRQ_TYPE_LEVEL_HIGH>;   /* signal irq */
 	};
 
-	ramoops_mem: ramoops_mem {
-		reg = <0x0 0x110000 0x0 0xf0000>;
-		reg-names = "ramoops_mem";
-	};
-
-	ramoops {
-		compatible = "ramoops";
-		record-size = <0x0 0x20000>;
-		console-size = <0x0 0x80000>;
-		ftrace-size = <0x0 0x00000>;
-		pmsg-size = <0x0 0x50000>;
-		memory-region = <&ramoops_mem>;
-	};
-
 	reserved-memory {
 		#address-cells = <2>;
 		#size-cells = <2>;
@@ -80,6 +66,15 @@
 			reg = <0x0 0x0 0x0 0x0>;
 		};
 
+		ramoops: ramoops@110000 {
+			compatible = "ramoops";
+			reg = <0x0 0x110000 0x0 0xf0000>;
+			record-size = <0x20000>;
+			console-size = <0x80000>;
+			ftrace-size = <0x00000>;
+			pmsg-size = <0x50000>;
+		};
+
 		/* global autoconfigured region for contiguous allocations */
 		linux,cma {
 			compatible = "shared-dma-pool";
diff --git a/arch/arm64/boot/dts/rockchip/rk3368-xikp.dtsi b/arch/arm64/boot/dts/rockchip/rk3368-xikp.dtsi
index a98790a5a990..0166406133c7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-xikp.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368-xikp.dtsi
@@ -68,20 +68,6 @@
 		};
 	};
 
-	ramoops_mem: ramoops_mem {
-		reg = <0x0 0x110000 0x0 0xf0000>;
-		reg-names = "ramoops_mem";
-	};
-
-	ramoops {
-		compatible = "ramoops";
-		record-size = <0x0 0x20000>;
-		console-size = <0x0 0x80000>;
-		ftrace-size = <0x0 0x00000>;
-		pmsg-size = <0x0 0x50000>;
-		memory-region = <&ramoops_mem>;
-	};
-
 	sdio_pwrseq: sdio-pwrseq {
 		compatible = "mmc-pwrseq-simple";
 		clocks = <&rk818 1>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
index 313844dc7f63..c8f68fa3a371 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
@@ -58,20 +58,6 @@
 		nvmem-cell-names = "id";
 	};
 
-	ramoops_mem: ramoops_mem {
-		reg = <0x0 0x110000 0x0 0xf0000>;
-		reg-names = "ramoops_mem";
-	};
-
-	ramoops {
-		compatible = "ramoops";
-		record-size = <0x0 0x20000>;
-		console-size = <0x0 0x80000>;
-		ftrace-size = <0x0 0x00000>;
-		pmsg-size = <0x0 0x50000>;
-		memory-region = <&ramoops_mem>;
-	};
-
 	fiq_debugger: fiq-debugger {
 		compatible = "rockchip,fiq-debugger";
 		rockchip,serial-id = <2>;
@@ -93,6 +79,15 @@
 			reg = <0x0 0x0 0x0 0x0>;
 		};
 
+		ramoops: ramoops@110000 {
+			compatible = "ramoops";
+			reg = <0x0 0x110000 0x0 0xf0000>;
+			record-size = <0x20000>;
+			console-size = <0x80000>;
+			ftrace-size = <0x00000>;
+			pmsg-size = <0x50000>;
+		};
+
 		secure_memory: secure-memory@20000000 {
 			compatible = "rockchip,secure-memory";
 			reg = <0x0 0x20000000 0x0 0x10000000>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-firefly-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-firefly-android.dts
index be4913ee6607..a26242da8b69 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-firefly-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-firefly-android.dts
@@ -63,20 +63,6 @@
 		nvmem-cell-names = "id";
 	};
 
-	ramoops_mem: ramoops_mem {
-		reg = <0x0 0x110000 0x0 0xf0000>;
-		reg-names = "ramoops_mem";
-	};
-
-	ramoops {
-		compatible = "ramoops";
-		record-size = <0x0 0x20000>;
-		console-size = <0x0 0x80000>;
-		ftrace-size = <0x0 0x00000>;
-		pmsg-size = <0x0 0x50000>;
-		memory-region = <&ramoops_mem>;
-	};
-
 	backlight: backlight {
 		compatible = "pwm-backlight";
 		pwms = <&pwm0 0 25000 0>;
@@ -173,6 +159,15 @@
 			compatible = "rockchip,stb-devinfo";
 			reg = <0x0 0x0 0x0 0x0>;
 		};
+
+		ramoops: ramoops@110000 {
+			compatible = "ramoops";
+			reg = <0x0 0x110000 0x0 0xf0000>;
+			record-size = <0x20000>;
+			console-size = <0x80000>;
+			ftrace-size = <0x00000>;
+			pmsg-size = <0x50000>;
+		};
 	};
 
 	rockchip-key {
-- 
2.35.3

